tidb/CHANGELOG.md

```diff
@@ -1,2 +1,10 @@
-# CHANGELOG - tidb
+# CHANGELOG - TiDB
 
+## 1.0.0 / 2021-01-25
+
+* [Improvement] Make user doc more understandable.
+* [Removed] Removed useless the monitor.
```

Comment:
```suggestion
* [Removed] Removed useless monitor.
```

Reply:
Does this just mean we removed a monitor that is unused, or is there another word missing from this sentence?